internal/operands/metrics/resources.go

```diff
@@ -268,9 +278,22 @@ func newPrometheusRule(namespace string) *promv1.PrometheusRule {
 			Groups: []promv1.RuleGroup{
 				{
 					Name:  "cnv.rules",
-					Rules: append(alertRulesList, getRecordRules()...),
+					Rules: append(getAlertRules(), getRecordRules()...),
 				},
 			},
 		},
 	}
 }
+
+func getRunbookURLTemplate() string {
+	runbookURLTemplate, exists := os.LookupEnv(runbookURLTemplateEnv)
+	if !exists {
+		runbookURLTemplate = defaultRunbookURLTemplate
+	}
+
+	if strings.Count(runbookURLTemplate, "%s") != 1 || strings.Count(runbookURLTemplate, "%") != 1 {
+		panic(errors.New("runbook URL template must have exactly 1 %s substring"))
```

Comment:
Can you return an error instead of panicking? The error is caused by user input and that could be handled.

Reply:
Updated, please let me know if the change makes sense